package.json

```diff
@@ -32,13 +32,18 @@
 		"clean": "rm -rf .next",
 		"dev": "next dev",
 		"build": "next build",
+		"prebuild-index": "bash scripts/run-indexer.sh -P 20123",
```

Comment:
Need some docs on how to run

Reply:
AFAIK, pre- and post- tasks are run automatically. `yarn build-index` will run `prebuild-index` before asnd `postbuild-index` after.